.github/workflows/prerelease.yml

```diff
@@ -156,7 +163,16 @@ jobs:
         shell: pwsh
         run: |
           build\windows\package_msi.ps1 -integration "$env:INTEGRATION" -arch ${{ matrix.goarch }} -tag "$env:TAG" -pfx_passphrase "$env:PFX_PASSPHRASE" -pfx_certificate_description "$env:PFX_CERTIFICATE_DESCRIPTION"
+      - name: Test win packages installation
+        uses: paologallinaharbur/test-packages-action/windows@add_win_test
+        with:
+          tag: ${{ env.TAG }}
+          integration: nri-${{ env.INTEGRATION }}
+          arch: ${{ matrix.goarch }}
+          upgrade: ${{ matrix.test-upgrade }}
       - name: Upload MSI to GH
+        # To avoid upload packages twice
+        if: startsWith(matrix.test-upgrade, 'false')
```

Comment:
does it need to be "startsWith"? just curious,  if testing for "equal" wouldn't work

Reply:
my first approach was to use `==` but is [was not working ](https://github.com/newrelic/nri-vsphere/runs/1763684685?check_suite_focus=true) . I don't [see](https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions) a function `equal` that would be clearer i agree.
I based on [this](https://github.community/t/matrix-cannot-be-used-in-jobs-level-if/17177/3) from the community